### DISCUSSION TITLE
Whats the different between select and the new selectv2 in beta

### DISCUSSION THREAD

**Discussion by thamibn** [element-plus/element-plus#3318] [Q&A] [answered]
What is the different between the old select component and the new one?

**Comment by sxzz** [comment, marked as answer]
V2 supports virtualized scroll. it's better for huge data. 
Please reference docs for more information: https://element-plus.org/#/zh-CN/component/select-v2

**Reply by thamibn** [reply]
so virtualized scroll means load more when your scrolling the list.
so if i understand you it will give us maybe 10 results and the more you scroll it loads the rest?

the docs dont say much currently thats why i havents seen any example that demostrate the differences.

**Reply by sxzz** [reply]
The normal `select`, will render all data to the DOM. It means if your have thounands or millions of items, the select will contains so many DOM element. so that it will make the page very stuck.

In contrast to select-v2, it just render the item you can see. Even if you have many data, it will just render items (maybe 10~20) that you can see.

**Reply by thamibn** [reply]
ohh i get you now thank you very much for your explanation.
